frontend/src/app/admin/settingsModal/page.tsx

```diff
@@ -181,6 +186,14 @@ export default function SettingsModal() {
                   Delete User
                 </button>
               )}
+              {user?.id === settingsUser.id && (
+                <button
+                  className={styles.deleteButton}
+                  onClick={handleRefreshSessionData}
+                >
+                  Refresh Session Data
+                </button>
+              )}
```

Comment:
Can't we put this button next to the Access Token button to avoid the extra `user?.id === settingsUser.id`?

Reply:
Because then the styling is more effort because i need another `<div>`